mozjs/src/lib.rs

```diff
@@ -46,6 +46,7 @@ pub mod jsapi {
 pub mod rust;
 
 mod consts;
+pub mod context;
```

Comment:
The file for this module appears to be missing.

Reply:
Oops. I'll fix that up tonight, hopefully.